xarray/core/common.py

```diff
@@ -556,6 +576,12 @@ def resample(self, freq=None, dim=None, how=None, skipna=None,
 
```

Comment:
Could you please delete the sentence starting with "Upsampling with filling is not supported"? This is no longer true :)

Reply:
Done